src/utils/curriculum/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const ENABLE_CYCLE_2 = false;
+export const ENABLE_CYCLE_2 = true;
```

Comment:
This should be `ENABLE_CYCLE_2 = false` so we don't accidentally release Cycle 2 early 😜